javascript/src/controllers/input-on-change-controller.js

```diff
@@ -0,0 +1,21 @@
+import { Controller } from 'stimulus'
+import { get } from '@rails/request.js'
+
+/**
+ * InputOnChange Controller
+ * Updates content based on a get request on input change
+ *
+ * <div data-controller="input-on-change" data-input-on-change-url-value="url_path">
+ *   <input data-action="input-on-change#change">
+ * </div>
```

Comment:
Incluye los atributos de **name** y **value** en el ejemplo porque sin ellos no funcionaría.

```suggestion
 * <div data-controller="input-on-change" data-input-on-change-url-value="/url/path">
 *   <input data-action="input-on-change#change" name="some-name" value="hola">
 * </div>
```